test/test_suit/test_schema_evolution_json_ignore_tombstone.py

```diff
@@ -65,7 +65,7 @@ def send(self):
                 key.append(json.dumps({'number': str(self.recordNum - 1)}).encode('utf-8'))
                 value.append(None)
                 key.append(json.dumps({'number': str(self.recordNum)}).encode('utf-8'))
-                value.append("") # community converters treat this as a tombstone
+                value.append("")  # community converters treat this as a tombstone
```

Comment:
TODO: add a new tests with higher row count so that we can have multiple batches